src/components/views/dialogs/devtools/ServerInfo.tsx

```diff
@@ -55,7 +55,7 @@ export async function getServerVersionFromFederationApi(client: MatrixClient): P
 
 const ServerInfo: React.FC<IDevtoolsProps> = ({ onBack }) => {
     const cli = useContext(MatrixClientContext);
-    const capabilities = useAsyncMemo(() => cli.getCapabilities(true).catch(() => FAILED_TO_LOAD), [cli]);
+    const capabilities = useAsyncMemo(() => cli.fetchCapabilities().catch(() => FAILED_TO_LOAD), [cli]);
```

Comment:
Why fetch over get here? Wouldn't cached be fine?

Reply:
I'm not sure to be honest, the code seems to think it's necessary though: I'm just aiming for maintaining the same functionality.

Reply:
oh yes, true